modules/afprog/afprog-grammar.ym

```diff
@@ -112,6 +113,7 @@ dest_afprogram_options
 dest_afprogram_option
 	: dest_writer_option
 	| dest_driver_option
+	| KW_KEEPALIVE '(' yesno ')'		{ ((AFProgramDestDriver *)last_driver)->keep_alive = $3; }
 	;
```

Comment:
please use a setter in this case, this is the preferred pattern over mucking within the private struct of the driver.

Also, keepalive is not the the best name, we are using keep-alive (e.g. with the dash in the socket module, so we should use the same).